app/Http/Controllers/ScholarPrePhdSeminarController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\PrePhdSeminar;
+use App\Models\Scholar;
+use App\Models\ScholarAppeal;
+use App\Types\RequestStatus;
+use App\Types\ScholarAppealTypes;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+
+class ScholarPrePhdSeminarController extends Controller
+{
+    public function request(Scholar $scholar)
+    {
+        $this->authorize('apply', [PrePhdSeminar::class, $scholar]);
+
+        return view('research.scholars.pre_phd_form', [
+            'scholar' => $scholar,
+        ]);
+    }
+
+    public function show(Scholar $scholar, PrePhdSeminar $appeal)
```

Comment:
I'd recommend not to use the word `appeal` here... It is adding a new term(since we no longer use it). It may cause confusions, later.